src/pcrlock/pcrlock.c

```diff
@@ -1927,6 +1927,9 @@ static int event_log_map_components(EventLog *el) {
                         continue;
                 }
 
+                if (!c->n_variants)
```

Comment:
hmm, in systemd we have this coding style requirement that arithmetic values (i.e. stuff that is neither boolean/boolean-like nor pointers) are compared with explicit comparisons against zero/non-zero. hence please write `if (c->n_variants == 0)` here